lib/net.js

```diff
@@ -534,15 +534,15 @@ Object.defineProperty(Socket.prototype, 'readyState', {
 });
 
 
-Object.defineProperty(Socket.prototype, 'bufferSize', {
-  get: function() { // eslint-disable-line getter-return
+defineProperty(Socket.prototype, 'bufferSize', {
+  get: function() {
```

Comment:
Well, you did remove the `eslint-disable` so I guess that's the reason for eslint error.